cmd/registry/cmd/check/rules/rule110/rule110.go

```diff
@@ -1,18 +1,18 @@
-// Copyright 2022 Google LLC
+// Copyright 2023 Google LLC. All Rights Reserved.
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Did you mean to change https to http?

Reply:
yes, this was inconsistent with all other notices